internal/controller/gameserver_controller.go

```diff
@@ -213,6 +213,7 @@ func (r *GameServerReconciler) reconcilePod(ctx context.Context, gameServer *gam
 	image := fmt.Sprintf("%s:%s", r.GameServerImage, gameServer.Spec.Version)
 
 	args := []string{}
+	args = append(args, gameServer.Spec.CmdArgs)
```

Comment:
i think this might need `append(args, gameServer.Spec.CmdArgs...)` to get the variadic form of `append`.

Reply:
ty